maintainers/maintainer-list.nix

```diff
@@ -4738,6 +4738,11 @@
     github = "tobimpub";
     name = "Tobias Mayer";
   };
+  tobiasBora = {
```

Comment:
Generally they want this in a separate commit.

Reply:
Ok, I created 3 commits, one for maintainer, one for the dependencies in perlpackage, one for the package.